frontend/src/concepts/pipelines/content/createRun/contentSections/ExperimentSection.tsx

```diff
@@ -14,54 +17,40 @@ type ExperimentSectionProps = {
 
 const ExperimentSection: React.FC<ExperimentSectionProps> = ({ value, onChange }) => {
   const { refreshAllAPI } = usePipelinesAPI();
-  const [{ items: experiments }] = useExperiments();
   const [openCreate, setOpenCreate] = React.useState(false);
-
-  const changeRef = React.useRef<ExperimentSectionProps['onChange']>(onChange);
-  changeRef.current = onChange;
-  React.useEffect(() => {
-    if (!value && experiments.length === 1) {
-      changeRef.current(experiments[0]);
-    }
-  }, [value, experiments]);
-
   return (
     <>
       <FormSection
-      // id={CreateRunPageSections.EXPERIMENT}
-      // title={runPageSectionTitles[CreateRunPageSections.EXPERIMENT]}
+        id={CreateRunPageSections.EXPERIMENT}
+        title={runPageSectionTitles[CreateRunPageSections.EXPERIMENT]}
       >
-        <Stack hasGutter>
-          <StackItem>
-            <SimpleDropdownSelect
-              placeholder="Select an experiment"
-              options={experiments.map((e) => ({ key: e.experiment_id, label: e.display_name }))}
-              value={value?.experiment_id ?? ''}
-              onChange={(id) => {
-                const experiment = experiments.find((p) => p.experiment_id === id);
-                if (experiment) {
-                  onChange(experiment);
-                }
-              }}
-            />
-          </StackItem>
-          <StackItem>
-            <Button variant="link" icon={<PlusCircleIcon />} onClick={() => setOpenCreate(true)}>
-              Create new experiment
-            </Button>
-          </StackItem>
-        </Stack>
+        {/* `minWidth` a temp fix for PF issue https://github.com/patternfly/patternfly/issues/6062
+      We can remove this after bumping to PF v5.2.0
+    */}
+        <FormGroup style={{ minWidth: 0 }}>
+          <Stack hasGutter>
+            <StackItem>
+              <ExperimentSelector selection={value?.display_name} onSelect={onChange} />
+            </StackItem>
+            <StackItem>
+              <Button variant="link" icon={<PlusCircleIcon />} onClick={() => setOpenCreate(true)}>
+                Create new experiment
+              </Button>
+            </StackItem>
+          </Stack>
+        </FormGroup>
       </FormSection>
-      <ManageExperimentModal
-        isOpen={openCreate}
-        onClose={(experiment) => {
-          if (experiment) {
-            refreshAllAPI();
-            onChange(experiment);
-          }
-          setOpenCreate(false);
-        }}
-      />
+      {openCreate && (
```

Comment:
Avoid doing this -- it causes weird modal issues. Open/Close via a prop

Reply:
We stopped doing this I think because it snapped the modal in -- so yes, css transitions. It's also not the way we have written our other dozens of modals... so why be different?

Reply:
There's no suggestion in my comment to be different, just asking for specific examples of your statement.

I suggest we continue to do things consistently and log issues to cover any incorrect usage to address across the app.

Reply:
resolved